src/containers/Qari/index.js

```diff
@@ -22,39 +21,61 @@ const styles = require('./style.scss');
   (state, ownProps) => ({
     surahs: state.surahs.entities,
     qari: state.qaris.entities[ownProps.params.id],
-    files: state.files.entities[ownProps.params.id]
+    files: state.files.entities[ownProps.params.id],
+    Playing: state.audioplayer.isPlaying,
   }),
-  { load }
+  { load, play, next, continuous}
 )
 export default class Qaris extends Component {
   static propTypes = {
     surahs: PropTypes.object.isRequired,
     qari: PropTypes.object.isRequired,
     files: PropTypes.object.isRequired,
-    load: PropTypes.func.isRequired
+    load: PropTypes.func.isRequired,
+    play: PropTypes.func.isRequired,
+    next: PropTypes.func.isRequired,
+    continuous: PropTypes.func.isRequired,
+    Playing: PropTypes.bool.isRequired
   };
 
+  handlePlayAll() {
```

Comment:
Could you make this into an arrow function, there won't be any need to `bind(this)` below with an arrow function.  Also to maintain consistency with the `handleSurahSelection` function